build/build.go

```diff
@@ -0,0 +1,177 @@
+package main
```

Comment:
I think this should live at the root with a go build that that skipa building it. It will still be runnable I think.

Reply:
if we move to root, we won't be able to have `package main`. and without `package main`, the main method won't run

Reply:
Is that true if the file is excluded from building with a nobuild directive?

Reply:
fair point.I have moved the file